editor/utils/formHelpers.ts

```diff
@@ -193,6 +182,48 @@ export const handleFormValueOutput = (
  */
 export const commonValidations = {
   isNumber: (value: string) => {
+    // Allow empty string for nullable fields
+    if (!value) return true;
     return /^\d+$/.test(value) || "This field must be a number";
+    return true;
+  },
+  isPositiveWholeNumber: (value: string) => {
+    // Allow empty string for nullable fields
+    if (!value) return true;
+    // Strip any non-numeric characters
+    const sanitizedValue = value.replace(/\D/g, "");
+
+    // If the original value contained a decimal, warn user that only whole numbers are accepted
+    if (value.includes(".")) {
+      return "Only whole numbers are accepted (no decimals)";
+    }
+
+    return (
+      /^\d+$/.test(sanitizedValue) ||
+      "This field must be a positive whole number"
+    );
+    return true;
+  },
+  isZipCode: (value: string) => {
+    // Allow empty string for nullable fields
+    if (!value) return true;
+
+    // If the value contains a decimal or non-numeric characters, provide specific feedback
+    if (value.includes(".")) {
+      return "Zip codes cannot contain decimal points";
+    }
+
+    if (/[^\d]/.test(value)) {
+      return "Zip codes can only contain numbers";
+    }
+
+    // Strip any non-numeric characters
+    const sanitizedValue = value.replace(/\D/g, "");
+
+    // Check for 5 digits
+    if (sanitizedValue.length !== 5) {
+      return `Zip code must be 5 digits (currently ${sanitizedValue.length})`;
+    }
+    return true;
```

Comment:
Should we accept valid zip+4s? For example, this is a valid USPS zip code: 
![image](https://github.com/user-attachments/assets/b971c859-adfa-46b1-baa8-28f81c2fae68)

Maybe a regex like this? `^\d{5}(-\d{4})?$`